### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gabrielmocanu**
Update README.md
This should be integrated after #85  and #86 .

Signed-off-by: Gabriel Mocanu <gabi.mocanu98@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,30 @@
-# iocla
-Repository public IOCLA\
-Contine scheletele de inceput pentru laboratoare si teme, precum si demo-urile
-de la cursuri.
+# IOCLA
 
-Il puteti descarca folosind comanda:
+This repository hosts the content and the source code for the `IOCLA` course.
+Contains all skeleton code for labs, homeworks and demos.
```
Comment (razvand) [inline README.md:4]:
```suggestion
It contains support documentation for all activities (labs, lectures, assignments), skeleton code for lab tasks and assignments and demo code used in lectures and labs.
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
+
+You can download it after you installed `git` on your system using:
```
Comment (razvand) [inline README.md:6]:
```suggestion
Use ``git`` to clone the repository locally:

```

File: README.md [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
 ```bash
 git clone https://github.com/systems-cs-pub-ro/iocla.git
 ```
-
-Inainte de inceputul fiecarui laborator si inainte sa incepeti sa implementati
-temele, e recomandat sa descarcat ultimele modificari aduse repository-ului
-folosind comanda:
+Before the start of each lab and before you star implementing homeworks,
```
Comment (razvand) [inline README.md:10]:
```suggestion

Make sure you have the latest version of the repository before working on any activity:

```

File: README.md [hunk 4]
```diff
@@ -1,21 +1,30 @@
...
+it is recommended to download the latest changes on the repository using:
 ```bash
 git pull
 ```
+## License
```
Comment (razvand) [inline README.md:15]:
```suggestion

## License
```

**Review by razvand — CHANGES_REQUESTED** [review]
I added some comments. I recommend we use assignments instead of homework.

**Comment by razvand** [comment]
@gabrielmocanu , please also add a commit description to pacify `checkpatch`.

**Comment by gabrielmocanu** [comment]
@razvand  Done.

**Pull request merged**